projects/schematics/src/ng-add/index.ts

```diff
@@ -66,7 +66,12 @@ function addModule(projectName?: string): Rule {
     });
     host.commitUpdate(recorder);
 
-    context.addTask(new TslintFixTask(modulePath, {}));
+    /* tslint is required to add a tslint fix task */
+    try {
+      require('tslint');
+      context.addTask(new TslintFixTask(modulePath, {}));
+    } catch (err) {
```

Comment:
[CI build](https://travis-ci.org/github/FortAwesome/angular-fontawesome/builds/709293853#L238) failed because of the empty block. It would be useful to inform user that formatting was skipped with something like below in the `catch` block:

```
      context.logger.warn('Formatting was skipped because tslint is not installed.');
```